dataland-frontend/tests/component/components/resources/dataTable/MultiLayerDataTableWithP2p.cy.ts

```diff
@@ -52,29 +52,29 @@ describe("Component test for P2pPanel", () => {
     getSectionHead("General").eq(1).click();
     getCellValueContainer("Data Date").should("be.visible");
 
-    getSectionHead("Ammonia").click();
-    getCellValueContainer("CCS technology adoption", 0, false).should("not.be.visible");
-    getSectionHead("Decarbonisation").click();
+    getSectionHead("Ammonia");
+    getCellValueContainer("CCS technology adoption", 0, false).should("be.visible");
+    getSectionHead("Decarbonisation");
     getCellValueContainer("CCS technology adoption")
       .should("contain.text", formatPercentageNumberAsString(ccsTechnologyAdoptionInPercent))
       .should("be.visible");
 
     getSectionHead("Livestock farming").click();
-    getSectionHead("Animal feed").click();
+    getSectionHead("Animal feed");
```

Comment:
can be deleted

Reply:
.should("exist") added according to comment below